semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -53,6 +53,32 @@ def parse(span_kind_value):
         return kind_map.get(span_kind_value)
 
 
+class InstrumentKind(Enum):
+    COUNTER = 1
+    UP_DOWN_COUNTER = 2
+    HISTOGRAM = 3
+    GAUGE = 4
+
+    @staticmethod
+    def parse(instrument_kind_value):
+        return InstrumentKind.kind_map().get(instrument_kind_value)
+
+    @staticmethod
+    def kind_map():
+        return {
```

Comment:
This should be a constant instead of being re-created at every call.

Reply:
removed this class and method